content/en/releases/version-skew-policy.md

```diff
@@ -239,3 +239,13 @@ or **{{< skew currentVersionAddMinor -3 >}}**)
 Running a cluster with `kube-proxy` instances that are persistently three minor versions behind
 `kube-apiserver` means they must be upgraded before the control plane can be upgraded.
 {{</ warning >}}
+
+## Controller clients and kube-apiserver upgrades
+
+Controller clients external to the kube-apiserver, such as kubelet, kube-controller-manager, and kube-scheduler, as well as aggregated API servers, must not have a higher version than the kube-apiserver they interact with to ensure compatibility and consistency within the cluster.
```

Comment:
Aggregated API servers can be easily configured to handle wide variations in skew between the library used to build the aggregated apiserver and the kube-apiserver.  The kube-apiserver levels that are supported by an aggregated apiserver belongs strictly to that aggregated apiserver, not the kubernetes authors.

Reply:
> The _default_ behavior of the project-provided library (k8s.io/apiserver) most aggregated servers are based on doesn't currently handle scenarios where it is running against an older kube-apiserver.

Is it _possible_ for an aggregated server author to modify those defaults in how they code things to make it work with a broader range? Probably, yes.

Is it _possible_ for a cluster administrator to modify options to make it work with a broader range? Maybe, depending on what options got exposed by the aggregated server author.

On reflection, I agree with @deads2k's statement that we should not take a position on what is or is not supported for an arbitrary aggregated API server here, since we don't determine how the server author is building their server and what skew they support.

Reply:
Okay, so does that mean the error was entirely from the user and their aggregated server?

Reply:
This was discussed at length today in the SIG meeting: https://www.youtube.com/watch?v=0TXm-DGcK1k, starting at minute 33:00

Reply:
Thanks, @fedebongio. 
I agree with @liggitt, this document isn't the right place to document this. I will be closing the PR.